app/uk/gov/hmrc/mobiletaxcreditssummary/domain/userdata/Child.scala

```diff
@@ -22,26 +22,34 @@ import java.time.{LocalDate, Period}
 import play.api.libs.json.{Json, OFormat}
 
 case class Child(
-  firstNames:    String,
-  surname:       String,
-  dateOfBirth:   LocalDate,
-  hasFTNAE:      Boolean,
-  hasConnexions: Boolean,
-  isActive:      Boolean,
-  dateOfDeath:   Option[LocalDate])
+  firstNames:     String,
+  surname:        String,
+  dateOfBirth:    LocalDate,
+  hasFTNAE:       Boolean,
+  hasConnections: Boolean,
+  isActive:       Boolean,
+  dateOfDeath:    Option[LocalDate])
 
 object Child {
   implicit val formats: OFormat[Child] = Json.format[Child]
 
   def getAge(child: Child): Long =
     Period.between(child.dateOfBirth, LocalDate.now).get(ChronoUnit.YEARS)
 
-  def getEligibleChildren(children: Seq[Child]): Seq[Person] =
+  def getEligibleChildren(children: Seq[Child]): Seq[Child] =
     children
       .filter { child =>
         getAge(child) < 20 &&
         child.isActive &&
         child.dateOfDeath.isEmpty
       }
-      .map(child => Person(forename = child.firstNames, surname = child.surname))
+
+  def hasFTNAEChildren(children: Seq[Child]): Boolean =
```

Comment:
Please can the casing convention be consistent, I prefer Pascal case so Ftnae instead of FTNAE